File: Syncfusion_Sample/Windows/Tools.Windows/Samples/2.0/Product Showcase/VS Demo/VB/VSDemoViewItems/ObjectBrowserView.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Windows.Forms

Namespace VSDemo
	''' <summary>
	''' Summary description for ObjectBrowser.
	''' </summary>
	Public Class ObjectBrowserView : Inherits System.Windows.Forms.UserControl
		Private panel1 As System.Windows.Forms.Panel
		Private browseBarItem As Syncfusion.Windows.Forms.Tools.XPMenus.ComboBoxBarItem
		Public objectBrowserToolBar As Syncfusion.Windows.Forms.Tools.XPMenus.XPToolBar
		Private sortNameParentBarItem As Syncfusion.Windows.Forms.Tools.XPMenus.ParentBarItem
		Private dockingImageList As System.Windows.Forms.ImageList
		Private sortTypeParentBarItem As Syncfusion.Windows.Forms.Tools.XPMenus.ParentBarItem
		Private backBarItem As Syncfusion.Windows.Forms.Tools.XPMenus.BarItem
		Private forwardBarItem As Syncfusion.Windows.Forms.Tools.XPMenus.BarItem
		Private findSymbolBarItem As Syncfusion.Windows.Forms.Tools.XPMenus.BarItem
		Private llObjBrowserBottom As System.Windows.Forms.LinkLabel
		Private components As System.ComponentModel.IContainer

		Public Sub New()
			' This call is required by the Windows.Forms Form Designer.
			InitializeComponent()

			' TODO: Add any initialization after the InitForm call

		End Sub

		''' <summary> 
		''' Clean up any resources being used.
		''' </summary>
		Protected Overrides Overloads Sub Dispose(ByVal disposing As Boolean)
			If disposing Then
				If Not components Is Nothing Then
					components.Dispose()
				End If
			End If
			MyBase.Dispose(disposing)
		End Sub

		#Region "Component Designer generated code"
		''' <summary> 
		''' Required method for Designer support - do not modify 
		''' the contents of this method with the code editor.
		''' </summary>
		Private Sub InitializeComponent()
			Me.components = New System.ComponentModel.Container()
			Dim resources As System.ComponentModel.ComponentResourceManager = New System.ComponentModel.ComponentResourceManager(GetType(ObjectBrowserView))
			Me.panel1 = New System.Windows.Forms.Panel()
			Me.llObjBrowserBottom = New System.Windows.Forms.LinkLabel()
			Me.objectBrowserToolBar = New Syncfusion.Windows.Forms.Tools.XPMenus.XPToolBar()
			Me.browseBarItem = New Syncfusion.Windows.Forms.Tools.XPMenus.ComboBoxBarItem()
			Me.sortNameParentBarItem = New Syncfusion.Windows.Forms.Tools.XPMenus.ParentBarItem()
			Me.dockingImageList = New System.Windows.Forms.ImageList(Me.components)
			Me.sortTypeParentBarItem = New Syncfusion.Windows.Forms.Tools.XPMenus.ParentBarItem()
			Me.backBarItem = New Syncfusion.Windows.Forms.Tools.XPMenus.BarItem()
			Me.forwardBarItem = New Syncfusion.Windows.Forms.Tools.XPMenus.BarItem()
			Me.findSymbolBarItem = New Syncfusion.Windows.Forms.Tools.XPMenus.BarItem()
			Me.panel1.SuspendLayout()
			Me.SuspendLayout()
			' 
			' panel1
			' 
			Me.panel1.Controls.Add(Me.llObjBrowserBottom)
			Me.panel1.Controls.Add(Me.objectBrowserToolBar)
			Me.panel1.Dock = System.Windows.Forms.DockStyle.Fill
			Me.panel1.Location = New System.Drawing.Point(0, 0)
			Me.panel1.Name = "panel1"
			Me.panel1.Size = New System.Drawing.Size(696, 88)
			Me.panel1.TabIndex = 0
			' 
			' llObjBrowserBottom
			' 
			Me.llObjBrowserBottom.BorderStyle = System.Windows.Forms.BorderStyle.FixedSingle
			Me.llObjBrowserBottom.Dock = System.Windows.Forms.DockStyle.Fill
			Me.llObjBrowserBottom.Font = New System.Drawing.Font("Calibri", 9.75F)
			Me.llObjBrowserBottom.LinkArea = New System.Windows.Forms.LinkArea(25, 58)
			Me.llObjBrowserBottom.LinkBehavior = System.Windows.Forms.LinkBehavior.AlwaysUnderline
			Me.llObjBrowserBottom.LinkColor = System.Drawing.Color.DarkGreen
			Me.llObjBrowserBottom.Location = New System.Drawing.Point(0, 25)
			Me.llObjBrowserBottom.Name = "llObjBrowserBottom"
			Me.llObjBrowserBottom.Size = New System.Drawing.Size(696, 63)
			Me.llObjBrowserBottom.TabIndex = 1
			Me.llObjBrowserBottom.TabStop = True
			Me.llObjBrowserBottom.Text = "public class RichTextBox: System.Windows.Forms.RichTextBox"
			Me.llObjBrowserBottom.UseCompatibleTextRendering = True
			' 
			' objectBrowserToolBar
			' 
			Me.objectBrowserToolBar.BackColor = System.Drawing.Color.Transparent
			Me.objectBrowserToolBar.Dock = System.Windows.Forms.DockStyle.Top
			Me.objectBrowserToolBar.Font = New System.Drawing.Font("Calibri", 9.75F)
			Me.objectBrowserToolBar.Items.AddRange(New Syncfusion.Windows.Forms.Tools.XPMenus.BarItem() { Me.browseBarItem, Me.sortNameParentBarItem, Me.sortTypeParentBarItem, Me.backBarItem, Me.forwardBarItem, Me.findSymbolBarItem})
			Me.objectBrowserToolBar.Location = New System.Drawing.Point(0, 0)
			Me.objectBrowserToolBar.Name = "objectBrowserToolBar"
			Me.objectBrowserToolBar.SeparatorIndices.AddRange(New Integer() { 1, 3})
			Me.objectBrowserToolBar.Size = New System.Drawing.Size(696, 25)
			Me.objectBrowserToolBar.Style = Syncfusion.Windows.Forms.VisualStyle.Office2007
			Me.objectBrowserToolBar.TabIndex = 0
			' 
			' browseBarItem
			' 
			Me.browseBarItem.ID = "Select "
			Me.browseBarItem.MinWidth = 100
			Me.browseBarItem.Text = "Select Component"
			' 
			' sortNameParentBarItem
			' 
			Me.sortNameParentBarItem.ImageIndex = 26
			Me.sortNameParentBarItem.ImageList = Me.dockingImageList
			Me.sortNameParentBarItem.ParentStyle = Syncfusion.Windows.Forms.Tools.XPMenus.ParentBarItemStyle.DropDown
			Me.sortNameParentBarItem.Style = Syncfusion.Windows.Forms.VisualStyle.OfficeXP
			' 
			' dockingImageList
			' 
			Me.dockingImageList.ImageStream = (CType(resources.GetObject("dockingImageList.ImageStream"), System.Windows.Forms.ImageListStreamer))
			Me.dockingImageList.TransparentColor = System.Drawing.Color.Transparent
			Me.dockingImageList.Images.SetKeyName(0, "")
			Me.dockingImageList.Images.SetKeyName(1, "")
			Me.dockingImageList.Images.SetKeyName(2, "")
			Me.dockingImageList.Images.SetKeyName(3, "")
			Me.dockingImageList.Images.SetKeyName(4, "")
			Me.dockingImageList.Images.SetKeyName(5, "")
			Me.dockingImageList.Images.SetKeyName(6, "")
			Me.dockingImageList.Images.SetKeyName(7, "")
			Me.dockingImageList.Images.SetKeyName(8, "")
			Me.dockingImageList.Images.SetKeyName(9, "")
			Me.dockingImageList.Images.SetKeyName(10, "")
			Me.dockingImageList.Images.SetKeyName(11, "")
			Me.dockingImageList.Images.SetKeyName(12, "")
			Me.dockingImageList.Images.SetKeyName(13, "")
			Me.dockingImageList.Images.SetKeyName(14, "")
			Me.dockingImageList.Images.SetKeyName(15, "")
			Me.dockingImageList.Images.SetKeyName(16, "")
			Me.dockingImageList.Images.SetKeyName(17, "")
			Me.dockingImageList.Images.SetKeyName(18, "")
			Me.dockingImageList.Images.SetKeyName(19, "")
			Me.dockingImageList.Images.SetKeyName(20, "")
			Me.dockingImageList.Images.SetKeyName(21, "")
			Me.dockingImageList.Images.SetKeyName(22, "")
			Me.dockingImageList.Images.SetKeyName(23, "")
			Me.dockingImageList.Images.SetKeyName(24, "")
			Me.dockingImageList.Images.SetKeyName(25, "")
			Me.dockingImageList.Images.SetKeyName(26, "")
			Me.dockingImageList.Images.SetKeyName(27, "")
			Me.dockingImageList.Images.SetKeyName(28, "")
			Me.dockingImageList.Images.SetKeyName(29, "")
			Me.dockingImageList.Images.SetKeyName(30, "")
			Me.dockingImageList.Images.SetKeyName(31, "")
			Me.dockingImageList.Images.SetKeyName(32, "")
			Me.dockingImageList.Images.SetKeyName(33, "")
			Me.dockingImageList.Images.SetKeyName(34, "")
			Me.dockingImageList.Images.SetKeyName(35, "")
			Me.dockingImageList.Images.SetKeyName(36, "")
			Me.dockingImageList.Images.SetKeyName(37, "")
			Me.dockingImageList.Images.SetKeyName(38, "")
			Me.dockingImageList.Images.SetKeyName(39, "")
			Me.dockingImageList.Images.SetKeyName(40, "")
			Me.dockingImageList.Images.SetKeyName(41, "")
			Me.dockingImageList.Images.SetKeyName(42, "")
			Me.dockingImageList.Images.SetKeyName(43, "")
			Me.dockingImageList.Images.SetKeyName(44, "")
			Me.dockingImageList.Images.SetKeyName(45, "")
			Me.dockingImageList.Images.SetKeyName(46, "")
			Me.dockingImageList.Images.SetKeyName(47, "")
			Me.dockingImageList.Images.SetKeyName(48, "")
			Me.dockingImageList.Images.SetKeyName(49, "")
			Me.dockingImageList.Images.SetKeyName(50, "")
			Me.dockingImageList.Images.SetKeyName(51, "")
			Me.dockingImageList.Images.SetKeyName(52, "")
			Me.dockingImageList.Images.SetKeyName(53, "")
			Me.dockingImageList.Images.SetKeyName(54, "")
			Me.dockingImageList.Images.SetKeyName(55, "")
			Me.dockingImageList.Images.SetKeyName(56, "")
			Me.dockingImageList.Images.SetKeyName(57, "")
			Me.dockingImageList.Images.SetKeyName(58, "")
			Me.dockingImageList.Images.SetKeyName(59, "")
			Me.dockingImageList.Images.SetKeyName(60, "")
			Me.dockingImageList.Images.SetKeyName(61, "")
			Me.dockingImageList.Images.SetKeyName(62, "")
			Me.dockingImageList.Images.SetKeyName(63, "")
			Me.dockingImageList.Images.SetKeyName(64, "")
			Me.dockingImageList.Images.SetKeyName(65, "")
			Me.dockingImageList.Images.SetKeyName(66, "")
			Me.dockingImageList.Images.SetKeyName(67, "")
			Me.dockingImageList.Images.SetKeyName(68, "")
			Me.dockingImageList.Images.SetKeyName(69, "")
			Me.dockingImageList.Images.SetKeyName(70, "")
			Me.dockingImageList.Images.SetKeyName(71, "")
			Me.dockingImageList.Images.SetKeyName(72, "")
			Me.dockingImageList.Images.SetKeyName(73, "")
			Me.dockingImageList.Images.SetKeyName(74, "")
			Me.dockingImageList.Images.SetKeyName(75, "")
			Me.dockingImageList.Images.SetKeyName(76, "")
			Me.dockingImageList.Images.SetKeyName(77, "")
			Me.dockingImageList.Images.SetKeyName(78, "")
			' 
			' sortTypeParentBarItem
			' 
			Me.sortTypeParentBarItem.ImageIndex = 36
			Me.sortTypeParentBarItem.ImageList = Me.dockingImageList
			Me.sortTypeParentBarItem.Style = Syncfusion.Windows.Forms.VisualStyle.OfficeXP
			' 
			' backBarItem
			' 
			Me.backBarItem.ImageIndex = 39
			Me.backBarItem.ImageList = Me.dockingImageList
			' 
			' forwardBarItem
			' 
			Me.forwardBarItem.ImageIndex = 40
			Me.forwardBarItem.ImageList = Me.dockingImageList
			' 
			' findSymbolBarItem
			' 
			Me.findSymbolBarItem.ImageIndex = 41
			Me.findSymbolBarItem.ImageList = Me.dockingImageList
			' 
			' ObjectBrowserView
			' 
			Me.Controls.Add(Me.panel1)
			Me.Name = "ObjectBrowserView"
			Me.Size = New System.Drawing.Size(696, 88)
			Me.panel1.ResumeLayout(False)
			Me.ResumeLayout(False)

		End Sub
		#End Region
	End Class
End Namespace

File: Syncfusion_Sample/Windows/Tools.Windows/Samples/2.0/Product Showcase/VS Demo/CS/VSDemoViewItems/ObjectBrowserView.cs
using System;
using System.Collections;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Windows.Forms;

namespace VSDemo
{
	/// <summary>
	/// Summary description for ObjectBrowser.
	/// </summary>
	public class ObjectBrowserView : System.Windows.Forms.UserControl
	{
		private System.Windows.Forms.Panel panel1;
		private Syncfusion.Windows.Forms.Tools.XPMenus.ComboBoxBarItem browseBarItem;
		public Syncfusion.Windows.Forms.Tools.XPMenus.XPToolBar objectBrowserToolBar;
		private Syncfusion.Windows.Forms.Tools.XPMenus.ParentBarItem sortNameParentBarItem;
		private System.Windows.Forms.ImageList dockingImageList;
		private Syncfusion.Windows.Forms.Tools.XPMenus.ParentBarItem sortTypeParentBarItem;
		private Syncfusion.Windows.Forms.Tools.XPMenus.BarItem backBarItem;
		private Syncfusion.Windows.Forms.Tools.XPMenus.BarItem forwardBarItem;
		private Syncfusion.Windows.Forms.Tools.XPMenus.BarItem findSymbolBarItem;
		private System.Windows.Forms.LinkLabel llObjBrowserBottom;
		private System.ComponentModel.IContainer components;

		public ObjectBrowserView()
		{
			// This call is required by the Windows.Forms Form Designer.
			InitializeComponent();

			// TODO: Add any initialization after the InitForm call

		}

		/// <summary> 
		/// Clean up any resources being used.
		/// </summary>
		protected override void Dispose( bool disposing )
		{
			if( disposing )
			{
				if(components != null)
				{
					components.Dispose();
				}
			}
			base.Dispose( disposing );
		}

		#region Component Designer generated code
		/// <summary> 
		/// Required method for Designer support - do not modify 
		/// the contents of this method with the code editor.
		/// </summary>
		private void InitializeComponent()
		{
            this.components = new System.ComponentModel.Container();
            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(ObjectBrowserView));
            this.panel1 = new System.Windows.Forms.Panel();
            this.llObjBrowserBottom = new System.Windows.Forms.LinkLabel();
            this.objectBrowserToolBar = new Syncfusion.Windows.Forms.Tools.XPMenus.XPToolBar();
            this.browseBarItem = new Syncfusion.Windows.Forms.Tools.XPMenus.ComboBoxBarItem();
            this.sortNameParentBarItem = new Syncfusion.Windows.Forms.Tools.XPMenus.ParentBarItem();
            this.dockingImageList = new System.Windows.Forms.ImageList(this.components);
            this.sortTypeParentBarItem = new Syncfusion.Windows.Forms.Tools.XPMenus.ParentBarItem();
            this.backBarItem = new Syncfusion.Windows.Forms.Tools.XPMenus.BarItem();
            this.forwardBarItem = new Syncfusion.Windows.Forms.Tools.XPMenus.BarItem();
            this.findSymbolBarItem = new Syncfusion.Windows.Forms.Tools.XPMenus.BarItem();
            this.panel1.SuspendLayout();
            this.SuspendLayout();
            // 
            // panel1
            // 
            this.panel1.Controls.Add(this.llObjBrowserBottom);
            this.panel1.Controls.Add(this.objectBrowserToolBar);
            this.panel1.Dock = System.Windows.Forms.DockStyle.Fill;
            this.panel1.Location = new System.Drawing.Point(0, 0);
            this.panel1.Name = "panel1";
            this.panel1.Size = new System.Drawing.Size(696, 88);
            this.panel1.TabIndex = 0;
            // 
            // llObjBrowserBottom
            // 
            this.llObjBrowserBottom.BorderStyle = System.Windows.Forms.BorderStyle.FixedSingle;
            this.llObjBrowserBottom.Dock = System.Windows.Forms.DockStyle.Fill;
            this.llObjBrowserBottom.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F);
            this.llObjBrowserBottom.LinkArea = new System.Windows.Forms.LinkArea(25, 58);
            this.llObjBrowserBottom.LinkBehavior = System.Windows.Forms.LinkBehavior.AlwaysUnderline;
            this.llObjBrowserBottom.LinkColor = System.Drawing.Color.DarkGreen;
            this.llObjBrowserBottom.Location = new System.Drawing.Point(0, 25);
            this.llObjBrowserBottom.Name = "llObjBrowserBottom";
            this.llObjBrowserBottom.Size = new System.Drawing.Size(696, 63);
            this.llObjBrowserBottom.TabIndex = 1;
            this.llObjBrowserBottom.TabStop = true;
            this.llObjBrowserBottom.Text = "public class RichTextBox: System.Windows.Forms.RichTextBox";
            this.llObjBrowserBottom.UseCompatibleTextRendering = true;
            // 
            // objectBrowserToolBar
            // 
            this.objectBrowserToolBar.BackColor = System.Drawing.Color.Transparent;
            this.objectBrowserToolBar.Dock = System.Windows.Forms.DockStyle.Top;
            this.objectBrowserToolBar.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F);
            this.objectBrowserToolBar.Items.AddRange(new Syncfusion.Windows.Forms.Tools.XPMenus.BarItem[] {
            this.browseBarItem,
            this.sortNameParentBarItem,
            this.sortTypeParentBarItem,
            this.backBarItem,
            this.forwardBarItem,
            this.findSymbolBarItem});
            this.objectBrowserToolBar.Location = new System.Drawing.Point(0, 0);
            this.objectBrowserToolBar.Name = "objectBrowserToolBar";
            this.objectBrowserToolBar.SeparatorIndices.AddRange(new int[] {
            1,
            3});
            this.objectBrowserToolBar.Size = new System.Drawing.Size(696, 25);
            this.objectBrowserToolBar.Style = Syncfusion.Windows.Forms.VisualStyle.Office2007;
            this.objectBrowserToolBar.TabIndex = 0;
            // 
            // browseBarItem
            // 
            this.browseBarItem.ID = "Select ";
            this.browseBarItem.MinWidth = 100;
            this.browseBarItem.Text = "Select Component";
            // 
            // sortNameParentBarItem
            // 
            this.sortNameParentBarItem.ImageIndex = 26;
            this.sortNameParentBarItem.ImageList = this.dockingImageList;
            this.sortNameParentBarItem.ParentStyle = Syncfusion.Windows.Forms.Tools.XPMenus.ParentBarItemStyle.DropDown;
            this.sortNameParentBarItem.Style = Syncfusion.Windows.Forms.VisualStyle.OfficeXP;
            // 
            // dockingImageList
            // 
            this.dockingImageList.ImageStream = ((System.Windows.Forms.ImageListStreamer)(resources.GetObject("dockingImageList.ImageStream")));
            this.dockingImageList.TransparentColor = System.Drawing.Color.Transparent;
            this.dockingImageList.Images.SetKeyName(0, "");
            this.dockingImageList.Images.SetKeyName(1, "");
            this.dockingImageList.Images.SetKeyName(2, "");
            this.dockingImageList.Images.SetKeyName(3, "");
            this.dockingImageList.Images.SetKeyName(4, "");
            this.dockingImageList.Images.SetKeyName(5, "");
            this.dockingImageList.Images.SetKeyName(6, "");
            this.dockingImageList.Images.SetKeyName(7, "");
            this.dockingImageList.Images.SetKeyName(8, "");
            this.dockingImageList.Images.SetKeyName(9, "");
            this.dockingImageList.Images.SetKeyName(10, "");
            this.dockingImageList.Images.SetKeyName(11, "");
            this.dockingImageList.Images.SetKeyName(12, "");
            this.dockingImageList.Images.SetKeyName(13, "");
            this.dockingImageList.Images.SetKeyName(14, "");
            this.dockingImageList.Images.SetKeyName(15, "");
            this.dockingImageList.Images.SetKeyName(16, "");
            this.dockingImageList.Images.SetKeyName(17, "");
            this.dockingImageList.Images.SetKeyName(18, "");
            this.dockingImageList.Images.SetKeyName(19, "");
            this.dockingImageList.Images.SetKeyName(20, "");
            this.dockingImageList.Images.SetKeyName(21, "");
            this.dockingImageList.Images.SetKeyName(22, "");
            this.dockingImageList.Images.SetKeyName(23, "");
            this.dockingImageList.Images.SetKeyName(24, "");
            this.dockingImageList.Images.SetKeyName(25, "");
            this.dockingImageList.Images.SetKeyName(26, "");
            this.dockingImageList.Images.SetKeyName(27, "");
            this.dockingImageList.Images.SetKeyName(28, "");
            this.dockingImageList.Images.SetKeyName(29, "");
            this.dockingImageList.Images.SetKeyName(30, "");
            this.dockingImageList.Images.SetKeyName(31, "");
            this.dockingImageList.Images.SetKeyName(32, "");
            this.dockingImageList.Images.SetKeyName(33, "");
            this.dockingImageList.Images.SetKeyName(34, "");
            this.dockingImageList.Images.SetKeyName(35, "");
            this.dockingImageList.Images.SetKeyName(36, "");
            this.dockingImageList.Images.SetKeyName(37, "");
            this.dockingImageList.Images.SetKeyName(38, "");
            this.dockingImageList.Images.SetKeyName(39, "");
            this.dockingImageList.Images.SetKeyName(40, "");
            this.dockingImageList.Images.SetKeyName(41, "");
            this.dockingImageList.Images.SetKeyName(42, "");
            this.dockingImageList.Images.SetKeyName(43, "");
            this.dockingImageList.Images.SetKeyName(44, "");
            this.dockingImageList.Images.SetKeyName(45, "");
            this.dockingImageList.Images.SetKeyName(46, "");
            this.dockingImageList.Images.SetKeyName(47, "");
            this.dockingImageList.Images.SetKeyName(48, "");
            this.dockingImageList.Images.SetKeyName(49, "");
            this.dockingImageList.Images.SetKeyName(50, "");
            this.dockingImageList.Images.SetKeyName(51, "");
            this.dockingImageList.Images.SetKeyName(52, "");
            this.dockingImageList.Images.SetKeyName(53, "");
            this.dockingImageList.Images.SetKeyName(54, "");
            this.dockingImageList.Images.SetKeyName(55, "");
            this.dockingImageList.Images.SetKeyName(56, "");
            this.dockingImageList.Images.SetKeyName(57, "");
            this.dockingImageList.Images.SetKeyName(58, "");
            this.dockingImageList.Images.SetKeyName(59, "");
            this.dockingImageList.Images.SetKeyName(60, "");
            this.dockingImageList.Images.SetKeyName(61, "");
            this.dockingImageList.Images.SetKeyName(62, "");
            this.dockingImageList.Images.SetKeyName(63, "");
            this.dockingImageList.Images.SetKeyName(64, "");
            this.dockingImageList.Images.SetKeyName(65, "");
            this.dockingImageList.Images.SetKeyName(66, "");
            this.dockingImageList.Images.SetKeyName(67, "");
            this.dockingImageList.Images.SetKeyName(68, "");
            this.dockingImageList.Images.SetKeyName(69, "");
            this.dockingImageList.Images.SetKeyName(70, "");
            this.dockingImageList.Images.SetKeyName(71, "");
            this.dockingImageList.Images.SetKeyName(72, "");
            this.dockingImageList.Images.SetKeyName(73, "");
            this.dockingImageList.Images.SetKeyName(74, "");
            this.dockingImageList.Images.SetKeyName(75, "");
            this.dockingImageList.Images.SetKeyName(76, "");
            this.dockingImageList.Images.SetKeyName(77, "");
            this.dockingImageList.Images.SetKeyName(78, "");
            // 
            // sortTypeParentBarItem
            // 
            this.sortTypeParentBarItem.ImageIndex = 36;
            this.sortTypeParentBarItem.ImageList = this.dockingImageList;
            this.sortTypeParentBarItem.Style = Syncfusion.Windows.Forms.VisualStyle.OfficeXP;
            // 
            // backBarItem
            // 
            this.backBarItem.ImageIndex = 39;
            this.backBarItem.ImageList = this.dockingImageList;
            // 
            // forwardBarItem
            // 
            this.forwardBarItem.ImageIndex = 40;
            this.forwardBarItem.ImageList = this.dockingImageList;
            // 
            // findSymbolBarItem
            // 
            this.findSymbolBarItem.ImageIndex = 41;
            this.findSymbolBarItem.ImageList = this.dockingImageList;
            // 
            // ObjectBrowserView
            // 
            this.Controls.Add(this.panel1);
            this.Name = "ObjectBrowserView";
            this.Size = new System.Drawing.Size(696, 88);
            this.panel1.ResumeLayout(false);
            this.ResumeLayout(false);

		}
		#endregion
	}
}
